src/content_state.ts

```diff
@@ -0,0 +1,61 @@
+import Logger from "./logging"
```

Comment:
This file should probably be called `state_content.ts`.

Reply:
Done, whoops.